lib/logging.dart

```diff
@@ -247,7 +245,7 @@ class Logger {
 }
 
 /// Handler callback to process log entries as they are added to a [Logger].
-typedef void LoggerHandler(LogRecord record);
+typedef LoggerHandler = void Function(LogRecord record);
```

Comment:
I don't think this is used anywhere in the package. We should deprecate this, too!

Reply:
I don't see any uses internally...

Reply:
Ah, and see also: https://github.com/dart-lang/logging/issues/13.